apps/fz_http/lib/fz_http/mailer/templates/auth_email/magic_link.html.heex

```diff
@@ -0,0 +1,15 @@
+<h3>Magic sign-in link</h3>
```

Comment:
Looks good. Thoughts on adding a `text/plain` template as well? I imagine some folks in our target market still use that.